docs/src/main/asciidoc/spring-cloud-sleuth.adoc

```diff
@@ -60,6 +57,123 @@ NOTE: Remember that tags are only collected and exported if there is a
 danger of accidentally collecting too much data without configuring
 something).
 
+NOTE: Currently the instrumentation in Spring Cloud Sleuth is eager - it means that
```

Comment:
I think it means more than this, correct? basically timing events are captured even when sleuth isn't exporting data to a tracing system.